app/open_ai_client.rb

```diff
@@ -28,22 +35,18 @@ def replace_placeholder(text, placeholder, value)
 
   def prompt
     @system_prompt
-    .gsub("${ROLE_PROMPT}", default_role_prompt)
-    .gsub("${INPUT_DESCRIPTION}", default_input_prompt)
-    .gsub("${USER_PROMPT}", default_user_prompt)
-    .gsub("${SUBMISSION}", "#{Submission.new.checklist}")
-    .gsub("${OUTPUT_DESCRIPTION}", default_output_prompt)
+    .gsub("@@SUBMISSION@@", "#{Submission.new.checklist}")
   end
 
   def system_prompt_default
 <<-SYSTEM_PROMPT
-#{ENV.fetch("ROLE_PROMPT", "${ROLE_PROMPT}")}
+#{@role_prompt}
 
-#{ENV.fetch("INPUT_DESCRIPTION", "${INPUT_DESCRIPTION}")}
+#{@input_description}
 
-#{ENV.fetch("USER_PROMPT", "${USER_PROMPT}")}
+#{@user_prompt}
 
-#{ENV.fetch("OUTPUT_DESCRIPTION", "${OUTPUT_DESCRIPTION}")}
+#{@output_description}
```

Comment:
Why would override these? 

The reason for this setup was to allow admins to add a `SYSTEM_PROMPT` with custom ordering of these keys;